src/ex2/ItemManager.js

```diff
@@ -0,0 +1,110 @@
+import PokemonClinet from "./PokemonClient.js";
+import UiLogic from "./Uilogic.js";
+
+class ItemManager {
+  constructor() {
+    this.taskList = [];
+    this.pokemonClinet = new PokemonClinet();
+    this.Orchestra = new UiLogic();
```

Comment:
match the convention of camel-case variables:
```suggestion
    this.orchestra = new UiLogic();
```